tool/language_color_generator.dart

```diff
@@ -8,17 +8,19 @@ const _path = './lib/src/const/language_color.dart';
 const _url = 'https://raw.githubusercontent.com/'
     'github/linguist/master/lib/linguist/languages.yml';
 
-Future main() async {
+Future<void> main() async {
   final response = await http.Client().get(_url);
 
   final yaml = loadYaml(response.body) as YamlMap;
+
   final stringBuffer = StringBuffer()
     ..writeln('// GENERATED CODE - DO NOT MODIFY BY HAND')
     ..writeln('// VERSION OF ${DateTime.now().toIso8601String()}')
     ..writeln()
-    ..writeln('const languagesColor = <String, String>{');
+    ..writeln('const kGitHubLanguageColors = <String, String>{');
```

Comment:
let's keep the name the same as above whatever we end up deciding
```suggestion
    ..writeln('const languageColors = <String, String>{');
```

Reply:
Yes!